src/JetBrains/Shared/Progress.cs

```diff
@@ -2,9 +2,10 @@
 using System.Diagnostics;
 using BenchmarkDotNet.Loggers;
 
-namespace BenchmarkDotNet.Diagnostics.dotTrace
+namespace BenchmarkDotNet.JetBrains.Shared
 {
-    public class Progress : IProgress<double>
+
+    internal sealed class Progress : IProgress<double>
```

Comment:
This was public before. Is it ok to remove it from public API? @AndreyAkinshin

Reply:
True, I forgot to mention that. I figured since it's not used by any configuration or anything it makes sense to not expose it? Users could have referenced it for some reason, but not because of any interaction with BenchmarkDotNet AFAICT.

If we keep it public I probably have to do something about the namespace, so that both packages don't expose the same type in the same namespace, unless that's not a big deal since we don't expect anyone to import `BenchmarkDotNet.JetBrains` regardless of which package is used?